code/core/src/common/js-package-manager/JsPackageManager.ts

```diff
@@ -195,6 +197,15 @@ export abstract class JsPackageManager {
     return result;
   }
 
+  #getIndent(filePath: string): string | number {
+    try {
+      const packageJson = JsPackageManager.getPackageJson(filePath);
+      return packageJson[Symbol.for('indent')];
```

Comment:
There is no need to use `Symbol.for`. Just define it as a constant at the top of the file and reference that.
```suggestion
      return packageJson[indentSymbol];
```

With at the top of the file:
```js
const indentSymbol = Symbol('indent');
```